### DISCUSSION TITLE
Missing Dotenv documentation page?

### DISCUSSION THREAD

**Discussion by KodyWiremane** [symfony/symfony#42678] [Q&A]
Hello,

There used to be a dedicated documentation page for the Dotenv component (e. g., [v4.3](https://symfony.com/doc/4.3/components/dotenv.html)), but for the current version [that page](https://symfony.com/doc/current/components/dotenv.html) is missing; instead, the _Documentation_ link leads to the [_Configuration_](https://symfony.com/doc/current/configuration.html) article that does not mention said component.

Um, shouldn't there be a dedicated Dotenv doc page for Symfony 5.3 as well?

**Comment by Seb33300** [comment]
Hello,

This is because the documentation for Dotenv has been moved to the configuration page: 
https://symfony.com/doc/current/configuration.html#configuring-environment-variables-in-env-files

**Reply by KodyWiremane** [reply]
That section looks more like about writing .env files, rather than using the actual Dotenv component.
_Configuration_ doesn't even mention `Dotenv` itself, not to speak of its methods (`load`, `overload`, `loadEnv`).

Not everyone uses the entire Symfony framework. If Dotenv is offered as a standalone component, it should have a standalone piece of documentation, shouldn't it? It still can refer to _Configuration_ to deduplicate .env-files explanation.

**Reply by Seb33300** [reply]
I don't remember where I have read it but as far as I remember, the documentation of standalone component usage was moved directly to the GitHub repository and the documentation on symfony.com now focus on the usage of component with the Symfony framework (to prevent confusion between both).

For example with Dotenv, you can find the previous documentation in the readme:
https://github.com/symfony/dotenv

I think @javiereguiluz can confirm if I am right

**Reply by KodyWiremane** [reply]
Yeah, but why not linking _that_ page as the doc page then? (And maybe linking _Configuration_ from that README) Or simply removing the _Docs_ link, coz it would be identical to _Code_ in that case; I can see some components are perfectly fine without _Docs_.

Following a _Docs_ link of a component, one expects to find docs on the _component_, precisely.
